backend/app/api/routes/calendar_routes.py

```
@@ -0,0 +1,45 @@
+                if isinstance(start, datetime) and start.year == year:
+                    # always return UTC midnight timestamp
+                    events.append({
+                        "date": start.strftime("%Y-%m-%dT00:00:00Z"),
+                        "summary": summary
+                    })
+                elif hasattr(start, "year") and start.year == year:
+                    events.append({
+                        "date": start.strftime("%Y-%m-%dT00:00:00Z"),
+                        "summary": summary
+                    })
+        return {"events": events}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)